src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -652,10 +655,10 @@ def get_observation(self, event: Event) -> pd.DataFrame:
             | (
                 (df_dependents["age"] < 24)
                 & (df_dependents["housing_type"] == "College")
-                & (df_dependents["income"] < 10_000)
+                & (df_dependents["last_year_income"] < 10_000)
```

Comment:
Refresh my memory - are we recording all possible dependents here, or the dependents they are actually claiming? A person could be eligible to be claimed by multiple filers.

Reply:
I think it is best to think of these as "eligible dependents", maybe I will change the var name to be clearer.  It is already a bit verbose, though.